src/IceRpc/Internal/ProtocolLoggerExtensions.cs

```diff
@@ -62,7 +62,7 @@ internal static partial class ProtocolLoggerExtensions
             remoteNetworkAddress);
 
     // Multiple logging methods are using same event id.
-#pragma warning disable SYSLIB1006
+#pragma warning disable SYSLIB1025
```

Comment:
The number was changed?

Reply:
Yes I get this new error with Visual Studio 2022 preview

Reply:
I think we should use 2 separate EventId here - for another PR!